drivers/SmartThings/jbl/src/discovery.lua

```diff
@@ -0,0 +1,104 @@
+local log = require "log"
+local discovery = {}

```

Comment:
nit: move the module declaration below the requires.

Reply:
Done